Index: src/main/web/actions/AuthActions.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/web/actions/AuthActions.js	(revision 06a8962e31fa30226d82a8a21075301f3f93f89f)
+++ src/main/web/actions/AuthActions.js	(date 1589174543000)
@@ -170,12 +170,13 @@
 };
 
 export const registerUser = (
-  fullName,
-  phone,
-  countryCode,
-  password,
-  thisRef,
-  callback,
+    fullName,
+    phone,
+    countryCode,
+    password,
+    thisRef,
+    gender,
+    callback,
 ) => {
   return dispatch => {
     dispatch({ type: REGISTER_USER });
@@ -191,6 +192,7 @@
         password: password,
         fullName: fullName,
         phone: phone,
+          gender: gender
       }),
     })
       .then(response => response.json())
Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- package.json	(revision 06a8962e31fa30226d82a8a21075301f3f93f89f)
+++ package.json	(date 1589105898000)
@@ -22,6 +22,7 @@
     "react-native-aws3": "^0.0.9",
     "react-native-camera": "^3.6.0",
     "react-native-card-stack-swiper": "^1.1.9",
+    "react-native-connectycube": "^3.0.0-beta.1",
     "react-native-country-picker-modal": "^0.8.0",
     "react-native-deck-swiper": "^1.6.7",
     "react-native-device-info": "^3.1.4",
@@ -41,6 +42,7 @@
     "react-native-onesignal": "3.4.1",
     "react-native-permissions": "^1.2.1",
     "react-native-progress": "^3.6.0",
+    "react-native-radio-buttons": "^1.0.0",
     "react-native-reanimated": "^1.3.0",
     "react-native-router-flux": "^4.0.6",
     "react-native-screens": "^2.4.0",
Index: src/main/web/components/NearbyUserDetail2.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/web/components/NearbyUserDetail2.js	(revision 06a8962e31fa30226d82a8a21075301f3f93f89f)
+++ src/main/web/components/NearbyUserDetail2.js	(date 1588567668000)
@@ -251,7 +251,7 @@
     }
 
     return (
-        <View style={{flex: 1, marginBottom: 20}}
+        <View style={{flex: 1}}
       >
         {/* Error modal */}
         <Modal
@@ -335,7 +335,7 @@
                         height: 400, //MC: Height of modal
                         alignItems: 'center',
                         justifyContent: 'center',
-                        backgroundColor: 'white',
+                        backgroundColor: 'rgba(',
                         // backgroundColor: 'rgba(0,0,0,0.1)',
                         // backgroundColor: 'transparent',
                         // backgroundColor: 'rgb(0, 255, 0, 1.0)',
Index: src/main/web/reducers/AuthReducer.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/web/reducers/AuthReducer.js	(revision 06a8962e31fa30226d82a8a21075301f3f93f89f)
+++ src/main/web/reducers/AuthReducer.js	(date 1589126436000)
@@ -34,6 +34,7 @@
   name: '',
   password: '',
   category: '',
+    gender: '',
   user: null,
   error: '',
   loading: false,
Index: src/main/web/components/RegistrationForm.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/web/components/RegistrationForm.js	(revision 06a8962e31fa30226d82a8a21075301f3f93f89f)
+++ src/main/web/components/RegistrationForm.js	(date 1589174543000)
@@ -35,6 +35,7 @@
 } from 'react-native-country-picker-modal';
 import Modal from 'react-native-modal';
 import {ScaledSheet} from 'react-native-size-matters';
+import {SegmentedControls} from 'react-native-radio-buttons'
 
 class RegistrationForm extends Component {
   constructor(props) {
@@ -58,6 +59,7 @@
     this.state = {
       cca2,
       callingCode,
+        gender: 'male',
       checked: false,
       data: {
         category: '',
@@ -122,6 +124,7 @@
       '+' + this.state.callingCode,
       password,
       this,
+        this.state.gender,
       function(success, thisRef) {
         if (success) {
           Actions.verificationCode({type: 'push'});
@@ -226,6 +229,10 @@
     this.setState({isVisible: false});
   }
 
+    onGenderChange(i) {
+        this.setState({gender: i});
+    }
+
   render() {
     return (
       <View style={{flex: 1, backgroundColor: 'white'}}>
@@ -395,6 +402,19 @@
 
               <Icon name="lock" color="#62cfb9" size={24} />
             </View>
+
+              <View style={[style.textInputContainer, {marginTop: 15}]}>
+                  <SegmentedControls
+                      options={['MALE', 'FEMALE']}
+                      onSelection={(gender) => this.onGenderChange(gender.toLowerCase())}
+                      selectedOption={(this.state.gender.toUpperCase())}
+                      optionContainerStyle={{flex: 1, borderWidth: 0}}
+                      tint={'#62cfb9'}
+                      selectedTint={'white'}
+                      backTint={'rgba(0,0,0,0.03)'}
+                      containerStyle={{flex: 1, borderWidth: 0}}
+                  />
+              </View>
 
             {Platform.OS === 'android' && (
               <View
