Index: src/main/web/components/ChatConversation.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/web/components/ChatConversation.js	(revision 99871e77ebb71362f720c817c7f7e4a01fcadc5b)
+++ src/main/web/components/ChatConversation.js	(date 1591545190000)
@@ -7,6 +7,7 @@
   Animated,
   StyleSheet,
   Dimensions,
+    Alert
 } from 'react-native';
 import {Actions} from 'react-native-router-flux';
 import {connect} from 'react-redux';
@@ -301,6 +302,27 @@
     Actions.popTo('myActivity');
   }
 
+    onBlock() {
+
+        Alert.alert(
+            'Are you sure you want to block this user?',
+            'You will no longer be able to communicate with this person',
+            [
+                {
+                    text: 'Yes I am sure',
+                    onPress: () => this.onBlockButtonPress()
+                },
+                {
+                    text: 'No, Cancel',
+                    onPress: () => console.log('Cancel Pressed'),
+                    style: 'cancel'
+                },
+            ],
+            {cancelable: false}
+        );
+    }
+
+
   onBlockButtonPress() {
     const {user} = this.props;
 
@@ -400,7 +422,7 @@
               {this.props.friend.fullName}
             </Text>
 
-            <TouchableOpacity onPress={this.onBlockButtonPress.bind(this)}>
+              <TouchableOpacity onPress={this.onBlock.bind(this)}>
               <Text
                 style={{
                   fontSize: 12,
